library/general/src/lib/config_file/config_file.rb

```diff
@@ -0,0 +1,29 @@
+module ConfigFile
+  class ConfigFile
```

Comment:
> \# A base class for models. Represents a configuration file as an object with domain-specific attributes/methods.
Uses
- a **file** (usually plain `::File`) to convert between the on-disk storage and a `String`,
- a **parser** (usually specific for each derived Model) to convert between that `String` and a private representation